sdk/core/core-http/src/webResource.ts

```diff
@@ -207,25 +232,56 @@ export class WebResource implements WebResourceLike {
     operationSpec: OperationSpec,
     response: HttpOperationResponse
   ) => undefined | OperationResponse;
+  /**
+   * Form data, used to build the request body.
+   */
   formData?: any;
+  /**
+   * Query added to the URL.
+   */
   query?: { [key: string]: any };
+  /**
+   * Specification of the HTTP request.
+   */
   operationSpec?: OperationSpec;
+  /**
+   * Whether to use a credential to sign the outgoing request.
```

Comment:
This actually has to do with an XHR setting: https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/withCredentials
```suggestion
   * Whether to send credentials (via cookies, authorization headers, or TLS client certificates) when making a request in the browser to a cross-site destination.
```